app/helpers/rota_helper.rb

```diff
@@ -0,0 +1,51 @@
+module RotaHelper
+  def select_allocated_organisations(rota, shift, slots)
+    organisation_uids = slots.select { |s| s.shift_id == shift.id }.map(&:organisation_uid)
+
+    organisations = @rota.organisations_with_uids(organisation_uids)
```

Comment:
Should this be referencing the rota instance variable? If it is, why are we passing a rota parameter down to it?